src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/http_client_full_default_options.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:framework="http://symfony.com/schema/dic/symfony"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:http_client>
+            <framework:default_options>
+                <framework:auth>foo:bar</framework:auth>
+                <framework:query>
+                    <framework:foo>bar</framework:foo>
+                    <framework:bar>baz</framework:bar>
+                </framework:query>
+                <framework:headers>
+                    <framework:X-powered>PHP</framework:X-powered>
```

Comment:
this should be made as `<framework:header name="X-powered">PHP</framework:header>`